app/components/WoWResults/FullScan/MobileDeltaTable.tsx

```
@@ -0,0 +1,193 @@
+            <tr className="text-gray-900 font-semibold dark:text-gray-100">
+              {columnList.map((col) => (
+                <th
+                  key={col.columnId}
+                  onClick={() => {
+                    if (col.columnId === columnSort) setDesc((state) => !state)
+                    else {
+                      setColumnSort(col.columnId)
+                      setDesc(true)
+                    }
+                  }}
+                  className={`py-2 px-3 sticky bg-gray-50 top-0 text-center cursor-pointer text-gray-900 dark:text-gray-100 dark:bg-gray-600`}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)